tests/chart_tests/test_houston_configmap.py

```diff
@@ -203,6 +203,39 @@ def test_houston_configmap_with_loggingsidecar_enabled():
     assert "vector" in prod_yaml["deployments"]["loggingSidecar"]["image"]
 
 
+def test_houston_configmap_with_loggingsidecar_enabled_with_index_prefix_overrides():
+    """Validate the houston configmap and its embedded data with
+    loggingSidecar."""
+    docs = render_chart(
+        values={
+            "global": {
+                "logging": {"indexNamePrefix": "astronomer"},
+                "loggingSidecar": {
+                    "enabled": True,
+                    "image": "quay.io/astronomer/ap-vector:0.32.2-1",
```

Comment:
We should make this test data something that is obviously test data, not something that could be mistaken for valid production values.
```suggestion
                    "image": "quay.io/astronomer/ap-vector:99.88.77",
```